src/tui.rs

```diff
@@ -483,31 +483,31 @@ impl ToipeTui {
         Ok(())
     }
 
-    // TODO: document this
+    /// Moves the cursor to the next char
     pub fn move_to_next_char(&mut self) -> MaybeError {
         let (x, y) = self.cursor_pos.next();
         write!(self.stdout, "{}", cursor::Goto(x, y))?;
 
         Ok(())
     }
 
-    // TODO: document this
+    /// Moves the cursor to the previous char
     pub fn move_to_prev_char(&mut self) -> MaybeError {
         let (x, y) = self.cursor_pos.prev();
         write!(self.stdout, "{}", cursor::Goto(x, y))?;
 
         Ok(())
     }
 
-    // TODO: document this
+    /// Moves the cursor to cur_pos
```

Comment:
This line doesn't seem particularly useful. Maybe something like this?
```suggestion
    /// Moves the cursor to just before the character to be typed next
```

Reply:
You're right, this describes it better.